scripts/var_props.py

```diff
@@ -989,13 +985,13 @@ def forward_transform(self, lvar_lname, rvar_lname, indices,
         # end if
         return f"{lhs_term} = {rhs_term}"
 
-    def reverse_transform(self, lvar_lname, rvar_lname, indices,
+    def reverse_transform(self, lvar_lname, rvar_lname, rvar_indices, lvar_indices,
                           adjust_hdim=None, flip_vdim=None):
         """Compute and return the the reverse transform from "var2" to "var1".
         <lvar_lname> is the local name of "var1".
         <rvar_lname> is the local name of "var2".
-        <indices> is a tuple of the loop indices for "var2" (i.e., "var2"
-           will show up in the RHS of the transform as "var2(indices)".
+        <rvar_indices> is a tuple of the loop indices for "var1" (i.e., "var1"
+           will show up in the RHS of the transform as "var1(rvar_indices)".
```

Comment:
Should add documentation for <lvar_indices> here

Reply:
done